app/models/business_unit.rb

```diff
@@ -38,9 +38,15 @@ class BusinessUnit < Team
            class_name: 'TeamsUsersRole',
            foreign_key: :team_id
 
-  has_many :managers, through: :manager_user_roles, source: :user
+  has_many :team_admin_user_roles,
+           -> { team_admin_roles  },
+           class_name: 'TeamsUsersRole',
+           foreign_key: :team_id
+
+           has_many :managers, through: :manager_user_roles, source: :user
```

Comment:
Tiny thing but is there a code formatting issue here (indention) or is it an artefact on Github?